typescript/infra/config/warp.ts

```diff
@@ -77,6 +76,19 @@ export async function getWarpConfig(
     envConfig,
     multiProvider,
   );
+  // Strip the owners from the router config
+  const routerConfigWithoutOwner = objMap(routerConfig, (_chain, config) => {
+    const { owner, ownerOverrides, ...configWithoutOwner } = config;
+    return configWithoutOwner;
+  });
+  // Isolate the owners from the router config
+  const abacusWorksEnvOwnerConfig = objMap(routerConfig, (_chain, config) => {
+    const { owner, ownerOverrides } = config;
+    return {
+      owner,
+      ownerOverrides,
+    };
+  });
```

Comment:
nit: could do a single reduce over routerConfig to avoid the double loop

Reply:
gonna keep as is just for readability / easier types